src/ui/pages/Backup/useRecoveryPhrase.ts

```diff
@@ -0,0 +1,27 @@
+import { useQuery } from '@tanstack/react-query';
+import { walletPort } from 'src/ui/shared/channels';
+import type { BackupContext } from './useBackupContext';
+
+async function getRecoveryPhrase(context: BackupContext) {
+  if (context.appMode === 'wallet') {
+    const mnemonic = await walletPort.request('getRecoveryPhrase', {
+      groupId: context.groupId,
+    });
+    return mnemonic.phrase;
+  } else {
+    return walletPort.request('getPendingRecoveryPhrase');
+  }
+}
+
+export function useRecoveryPhrase(context: BackupContext) {
+  return useQuery({
+    queryKey: ['getRecoveryPhrase', context],
+    queryFn: () => getRecoveryPhrase(context),
+    suspense: false,
+    retry: 0,
+    cacheTime: 0,
+    refetchOnMount: false,
+    refetchOnWindowFocus: true,
```

Comment:
why?

Reply:
I only added `cacheTime: 0`, everything else is unchanged, I didn't refactor this part

Reply:
but yes, it doesn't make sense :) I'll remove that